lib/meandro/rules/unused_macros.ex

```diff
@@ -16,7 +16,7 @@ defmodule Meandro.Rule.UnusedMacros do
   end
 
   @impl Meandro.Rule
-  def is_ignored?({module, macro}, {module, macro}) do
+  def is_ignored?({macro, arity}, {macro, arity}) do
```

Comment:
What about an extra pattern, like…
```elixir
def is_ignored?({macro, _arity}, macro), do: true
```
…to ignore all versions of a certain macro by name.